src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,9 @@
+/* Catalog Start */
+.catalog {
+  margin: 0 auto;
+  padding: 50px calc((100% - (236px * 4)) / 2);
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
Instead of repeating '1fr' 4 times, you can use the 'repeat()' function: grid-template-columns: repeat(4, 1fr);